public/app.d.ts

```diff
@@ -3019,5 +3019,78 @@ declare namespace JMap {
        */
       function display(message: string, options?: JAppMessageOptions): void
     }
+
+    /**
+     * **JMap.Application.Favorite**
+     * 
+     * You can manage everything related to your favorites from here.
+     */
+    namespace Favorite {
+      /**
+       * **JMap.Application.Favorite.addFavorite**
+       * 
+       * Return the favorite created using the location or nothing if there is an error.
+       * 
+       * @param newLocation the location of the new favorite
+       * @example ```ts
+       * 
+       * const newLocation: JLocation = {
+       *  x: 50,
+       *  y: 50
+       * }
+       * JMap.Application.Favorite.add(newLocation)
+       * ```
+       */
+      function add(newLocation: JLocation): JAppFavorite|void
+
+      /**
+       * **JMap.Application.Favorite.deleteById**
+       * 
+       * Delete the favorite of the given id from the JMap Favorite User menu.
+       *       
```

Comment:
Ajoute des @throw pour dire que ta méthodes va lever une exception, ici par exemple :
@throw if the given favorite is not found

Reply:
Ajoute des @throw dans tes autres methodes stp